Transcelerator/NewQuestionDlg.cs

```diff
@@ -212,21 +202,46 @@ public TranslatablePhrase BasePhrase
 			m_scrPsgReference.PassageChanged += r =>
 			{

```

Comment:
I'm guessing it was done to avoid having this event fire when the control was first initialized. I vaguely assumed this was somehow related to the comment about overcoming the HACK in ScrPassageControl (to prevent setting m_existingStartRef prematurely?), but maybe not.